exercises/etl/canonical-data.json

```diff
@@ -0,0 +1,62 @@
+{
+  "transform": {
+    "description": "transforms the input map from a map of value: [ keys ] to a map of key: value for each of the keys",
+    "cases": [
+      {
+        "description": "transforms one value",
+        "input": {
+          "1": ["A"]
+        },
+        "expected": {
+          "a" : 1
+        }
+      },
+      {
+        "description": "transforms more values",
+        "input": {
+          "1": ["A", "E", "I", "O", "U" ]
+        },
+        "expected": {
+          "a" : 1,
+          "e" : 1,
+          "i" : 1,
+          "o" : 1,
+          "u" : 1
+        }
+      },
+      {
+        "description": "transforms more keys",
```

Comment:
Though it could be an interesting error/edge scenario, it doesn't fit in with the problem description which says we're dealing with scrabble tile scores.  Because of scrabble rules, there is no case where one A tile is worth 1 and another is worth 2.